easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -53,27 +53,24 @@
 def det_pylibdir(plat_specific=False):
     """Determine Python library directory."""
     log = fancylogger.getLogger('det_pylibdir', fname=False)
-    pyver = get_software_version('Python')
-    if not pyver:
-        raise EasyBuildError("Python module not loaded.")
-    else:
-        # determine Python lib dir via distutils
-        # use run_cmd, we can to talk to the active Python, not the system Python running EasyBuild
-        prefix = '/tmp/'
-        args = 'plat_specific=%s, prefix="%s"' % (plat_specific, prefix)
-        pycmd = "import distutils.sysconfig; print(distutils.sysconfig.get_python_lib(%s))" % args
-        cmd = "python -c '%s'" % pycmd
-        out, ec = run_cmd(cmd, simple=False)
-        txt = out.strip().split('\n')[-1]
-
-        # value obtained should start with specified prefix, otherwise something is very wrong
-        if not txt.startswith(prefix):
-            raise EasyBuildError("Last line of output of %s does not start with specified prefix %s: %s (exit code %s)",
-                                 cmd, prefix, out, ec)
-
-        pylibdir = txt[len(prefix):]
-        log.debug("Determined pylibdir using '%s': %s", cmd, pylibdir)
-        return pylibdir
+
+    # determine Python lib dir via distutils
+    # use run_cmd, we can to talk to the active Python, not the system Python running EasyBuild
```

Comment:
shouldn't we at least print out a debugging message about which python is being used?